cmd/seccomp-operator/main.go

```diff
@@ -103,6 +112,7 @@ func run(*cli.Context) error {
 
 	ctrlOpts := ctrl.Options{
 		SyncPeriod: &sync,
+		Scheme:     scheme,
```

Comment:
nitpick: you can leave this empty and `ctrl.NewManager()` will create a `*runtime.Scheme` for you, which you can then add types to with something like:
```
if err := seccompoperatorv1alpha1.AddToScheme(mgr.GetScheme()); err != nil {
	return errors.Wrap(err, "Cannot add core Seccomp APIs to scheme")
}
```